docs/advanced/conditional_variables.rst

```diff
@@ -0,0 +1,29 @@
+.. _conditional_variables:
+
+Using conditional variables
+---------------------------
+
+You can specify conditional variables that will only be rendered if a certain other variable is set to 'yes' and the name starts with _if_use_::
```

Comment:
The term ``rendered`` is a bit confusing here. At least I did not get what it means from this sentence. The explanation below then tells, but anyway, the sentence here should be understandable on its own.

Reply:
I just read it first time - everything clear.